### ISSUE TITLE
Could not create notification contract definition, provider returned status code 404

### ISSUE THREAD

**Issue by tuanphampmt** [eclipse-tractusx/item-relationship-service#310]
When I run to the IRS, I get the error `ERROR [main] o.e.t.i.e.s.EdcRegistration: Could not create notification contract definition, provider returned status code 404`

The reason is that the data body is used in the wrong format
API: /management/v3/assets
Data body: `{
    "@context": {},
    "asset": {
        "@id": "ess-response-asset",
        "properties": {
            "description": "ESS notification endpoint",
            "contenttype": "application/json",
            "notificationtype": "ess-supplier-response",
            "notificationmethod": "receive"
        }
    },
    "dataAddress": {
        "baseUrl": "http://irs:8080/ess/notification/receive",
        "type": "HttpData",
        "proxyBody": "true",
        "proxyMethod": "true"
    }
}`
Call API /management/v3/assets error: mandatory object 'https://w3id.org/edc/v0.0.1/ns/properties' is missing

**Comment by tuanphampmt** [comment]
Hope you can help me resolve this issue

**Comment by jzbmw** [comment]
The IRS is currently using the EDC V2 management endpoint. 
@ds-jhartmann will be adding some information.

**Comment by ds-jhartmann** [comment]
Hi @tuanphampmt, as Johannes mentioned, we still use the /management/v2 endpoint for our EDC communication.
We plan on updating this in one of our upcoming sprints. 
In the meantime you can switch to the /v2/ management endpoint for the IRS. This endpoint is still supported in EDC 0.5.3 and works with IRS 4.0.0+

**Comment by jzbmw** [comment]
see #313

**Comment by ds-jhartmann** [comment]
Hi @tuanphampmt, 
we updated our EDC implementation to fully replace all the deprecated endpoints of EDC 0.5.3. 
This is available with the latest IRS version [4.3.0](https://github.com/eclipse-tractusx/item-relationship-service/releases/tag/4.3.0).
The configuration has been updated to use the "management/v3/assets" endpoint by default. You can configure this via helm chart:
```yaml
ess:
  edc:
    host: <your-edc-base-url>
  assetsPath: /management/v3/assets
  policydefinitionsPath: /management/v2/policydefinitions
  contractdefinitionsPath: /management/v2/contractdefinitions
```
I will close this issue now. Feel free to reopen it, if you still have issues.
